cryptography/build.gradle.kts

```diff
@@ -91,3 +93,17 @@ kotlin {
         }
     }
 }
+
+fun registerCopyTestResourcesTask(target: String) {
+    val task = tasks.register<Copy>("copy${target.capitalized()}TestResources") {
+        from("src/commonTest/resources")
+        into("build/bin/$target/debugTest/resources")
+    }
+    tasks.findByName("${target}Test")?.dependsOn(task)
+}
+
+// TODO can we avoid manually listing the native targets?
```

Comment:
😞  I was having a look to this, and couldn't come up with a way to identify this iOS native targets... I think we don't have an easy way out of this

Reply:
They iOS targets are set in `Multiplatform.kt` maybe we could expose a `darwinTargets` property?

Reply:
That could work too 👍  I think if we can reuse it later, definitely is a good spot to be placed